pkg/config/config.go

```diff
@@ -765,11 +765,19 @@ func addConfigs(dirPath string, configs []string) ([]string, error) {
 // Returns the list of configuration files, if they exist in order of hierarchy.
 // The files are read in order and each new file can/will override previous
 // file settings.
-func systemConfigs() ([]string, error) {
-	var err error
-	configs := []string{}
-	path := os.Getenv("CONTAINERS_CONF")
-	if path != "" {
+func systemConfigs() (configs []string, err error) {
+	if path := os.Getenv("CONTAINERS_CONF_OVERRIDE"); path != "" {
+		if _, err := os.Stat(path); err != nil {
+			return nil, fmt.Errorf("CONTAINERS_CONF_OVERRIDE file: %w", err)
+		}
+		// Add the override config last to make sure it can override any
+		// previous settings.
+		defer func() {
+			configs = append(configs, path)
```

Comment:
I think there should be an 

if err == nil check here. Otherwise we could return a configs entry even when an error happens.

Reply:
Can you elaborate on that?

The error is being checked in line 770 above.  The condition in this line is that 1) _OVERRIDE is set and 2) that the path exists.

Reply:
CONTAINERS_CONF_OVERRIDE=/PATHTO/containers.conf CONTAINERS_CONF=/PATHTO/bogus.conf ...

Will return `configs=[]string{"/PATHTO/containers.conf"}, err="/PATHTO/bogus.conf FILE DOES NOT EXISTS"`
 
Where I think it would be safer to return

              `configs=nil,  err="/PATHTO/bogus.conf FILE DOES NOT EXISTS"`

Reply:
Thanks!  Will update the code :+1: